__tests__/app.test.js

```diff
@@ -37,4 +37,39 @@ describe("GET/api/topics",()=>{
               })
           })
       })
-  })  
\ No newline at end of file
+  })  
+
+  describe('4. GET /api/articles',()=>{
+    test('returns 200 with all articles data',()=>{
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((data)=>{
+            expect(data.body.article).toBeInstanceOf(Array);
+            data.body.article.forEach((element)=>{
```

Comment:
can we add a check for the length to be greater than zero/equal to what we expect here too to avoid any possibility of false positives?